### PR TITLE
core-svc-chore(logs): error logs improvement

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
## Overview
- error logs improvement

## Changes
- `News` module
- `Publication` module
- `Event` module

## Evidence:
project: Portal Jabar
title: core-svc-chore(logs): error logs improvement
participants: @rachadiannovansyah @ayocodingit

### CODE REVIEW

File: core-service/src/modules/master-data-publication/delivery/http/publication_handler.go [hunk 1]
```diff
@@ -227,6 +232,8 @@ func (h *MasterDataPublicationHandler) Update(c echo.Context) (err error) {
 
 	err = h.MdpUcase.Update(ctx, body, ID)
 	if err != nil {
+		log.AdditionalInfo["publication_error_updated"] = err.Error()
+		h.Logger.Error(log, err)
```
Comment (ayocodingit) [inline core-service/src/modules/master-data-publication/delivery/http/publication_handler.go:236]: For message error, better create on message on the params, so we dont create key/field new on elastic

I mean for example

```go
h.Logger.Error(log, err.Error())

```

How do you think?

Comment (rachadiannovansyah) [reply core-service/src/modules/master-data-publication/delivery/http/publication_handler.go:236]: it nice question but in this case we need to count the specific error on each process of the module. in this step, we write to `publication_error_updated` attribute to count the error of the publication update process.

instead we use `h.Logger.Error(log, err.Error()) all error will collected on the same one. @ayocodingit

Comment (ayocodingit) [reply core-service/src/modules/master-data-publication/delivery/http/publication_handler.go:236]: Can you filter on elastic by uri or full path for specific location, so this message error still category router

Comment (rachadiannovansyah) [reply core-service/src/modules/master-data-publication/delivery/http/publication_handler.go:236]: As you know, the logger for `Error` function is required to be an `error` type not as a string.

```
func (l *Logrus) Info(logsField *_goLog.LoggerData, message string) {
	l.logger.Info(logsField, message)
}

func (l *Logrus) Error(logsField *_goLog.LoggerData, e error) {
	l.logger.Error(logsField, e)
}
```

so we cannot send `err.Error()` as params of the `h.Logger.Error(log, err.Error())`

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**